packages/arb-token-bridge-ui/src/pages/_document.tsx

```diff
@@ -1,18 +1,35 @@
 import { Html, Head, Main, NextScript } from 'next/document'
 
+export const siteTitle = 'Bridge to Arbitrum'
+const siteDomain = 'https://bridge.arbitrum.io'
+const siteDescription =
+  'Built to scale Ethereum, Arbitrum brings you 10x lower costs while inheriting Ethereum’s security model. Arbitrum is a Layer 2 Optimistic Rollup.'
+
 export default function Document() {
   return (
     <Html lang="en">
       <Head>
         <meta charSet="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
This should not be in `_document`:
`Warning: viewport meta tags should not be used in _document.js's <Head>. https://nextjs.org/docs/messages/no-document-viewport-meta`